### ISSUE TITLE
Outline claims sometimes more and more memory and the page gets unresponsive

### ISSUE THREAD

**Issue by jontze** [outline/outline#5540]
When I open outline I can see that the memory usage increases over a relatively short time and the page gets unresponsive.

**To Reproduce**
1. Go to https://<some_instance>.getoutline.com
2. Click on a document. I could reproduce it multiple times when going to the default `What is outline`-document
3. Wait a little bit and investigate the ram and CPU usage of the chrome process
4. Chrome might report that the page is unresponsive

**Expected behavior**
RAM should not get filled while doing nothing. Chrome should not report strange behavior.

**Screenshots**

**Chrome reporting that the page is unresponsive**
![image](https://github.com/outline/outline/assets/42588836/8ac41985-2696-4500-ac19-1246c96e661d)

**Ram usage before I killed the tab**
![image](https://github.com/outline/outline/assets/42588836/a9fde92f-6ad4-4254-a237-20cb3bfd16a4)

**Ram after I killed the tab**
_Edit: I marked the wrong process in the screenshot. The process of the outline tab is obvs gone as the tab was closed_
![image](https://github.com/outline/outline/assets/42588836/18efa62c-358f-4610-b34a-09603be0d6b1)

**Outline (please complete the following information):**
- Install: The SaaS trial on getoutline.com
- Version: I assume this might be v0.70.1 to the time of writing this.

**Desktop (please complete the following information):**
 - OS: linux arch
 - Browser: Google chrome
 - Version `Google Chrome 114.0.5735.198`

**Comment by tommoor** [comment]
Would you be able to try again in a tab without any browser extensions running? We haven't had any other reports of memory leaks like this I also keep many copies of the app running all day…

**Comment by jontze** [comment]
The hint with the browser extensions was a good one! 

Indeed everything worked perfectly fine when I run the app in an incognito window without any extensions. After that I enabled step for step my extensions again and could find out that issue is the [darkreader extension](https://github.com/darkreader/darkreader). As soon as I enabled this one, CPU usage goes up and the memory usage of the tab gets more and more.

So not an issue with outline at all, just disable the dark reader. Outline syncs with system preferences either way :laughing: 

Thanks for the quick help! I hope this issue might help others as well^^

_Seems to be a similar problem as reported in https://github.com/darkreader/darkreader/issues/10691_
